cadence/gateway/cadence/client.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Tuple
+import grpc
+import uber.cadence.api.v1.service_domain_pb2 as service_domain_pb2
+import uber.cadence.api.v1.service_domain_pb2_grpc as service_domain_pb2_grpc
+
+
+class WorkflowClient(object):
```

Comment:
I think the right way to do it is to modify this `WorkflowService` to support gRPC in addition to TChannel. 

This will make it much simpler for you to do other migration

Reply:
This is a good call and I was considering as one of the approach. One question before changing it there, do you want me to change the original TChannel method for the grpc one?

The reason I did it on a separate class was to, when I had all the methods ready, to replace them all at once (and roll back if case we had to).

 But I'm happy replacing the method right now :)

Reply:
I think we should allow passing some parameters to choose using tchannel or grpc, and default to tchannel to be compatible